apollo-runtime/src/main/java/com/apollographql/apollo/cache/ApolloCacheHeaders.java

```diff
@@ -0,0 +1,21 @@
+package com.apollographql.apollo.cache;
+
+import com.apollographql.apollo.cache.normalized.NormalizedCache;
+
+/**
+ * A collection of cache headers that Apollo's implementations of {@link NormalizedCache} respect.
+ */
+public final class ApolloCacheHeaders {
+
+  /**
+   * Records from this request should not be stored in the {@link NormalizedCache}.
+   * This does not specify that a request should not be read from a cache. To control where
+   * a response is read use {@link com.apollographql.apollo.cache.normalized.CacheControl}.
+   */
+  public static final String NO_CACHE = "no-cache";
```

Comment:
Can we come up with better naming, as without looking into JavaDoc the name confusing, does it mean it won't use cache at all for both reading/writing.